src/fish/install.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 FISHER=${FISHER:-"true"}
+FISHER_PLUGINS=${FISHER_PLUGINS:-""}
```

Comment:
```suggestion
FISHERPLUGINS=${FISHERPLUGINS:-""}
```

Reply:
I propose to go with the same approach maintainers currently use in https://github.com/devcontainers/features: https://github.com/devcontainers/spec/issues/57#issuecomment-1309015728

```bash
BASH_ARRAY=( ${COMMA_SEPARATED_OPTION//,/ } )
```